_team/bryan-pan.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Bryan Pan
+year: 2022
+major: Computer Science
+---
+
+[!A beautiful image of Bryan Pan](../public/team/bryan-pan.png)
```

Comment:
If you're going to use team images elsewhere (e.x. a "team" page), would you want to front matter this instead?

Reply:
It would be nice to support images from inside the markdown itself. This is more of an example of how to utilize it. But yes we should add it to the frontmatter.. I'll add it to this example.

Reply:
actually, i thought about this again and I think it might be better to lint the front matter to make sure there exists an asset that has the same slug

For example:
```
root
├── _team/
│   ├── bryan-pan.md
│   └── ...
├── public/
│   ├── team/
│   │   ├── bryan-pan.png
│   │   └── ...
│   └── ...
└── ...
```

Reply:
Seems reasonable to me and is pretty low-friction, we'd just need to enforce a subset (or a single) file extension, right?

I was working on something similar for Teach LA before my term ended, "linting" images by filename, aspect ratio, etc.